app/src/main/java/org/mifos/mobile/ui/activities/PassCodeActivity.java

```diff
@@ -51,10 +55,24 @@ public void startNextActivity() {
 
     @Override
     public void startLoginActivity() {
-        Intent i = new Intent(PassCodeActivity.this, LoginActivity.class);
-        i.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
-        startActivity(i);
-        finish();
+        AlertDialog.Builder builder = new AlertDialog.Builder(this);
+        builder.setMessage(R.string.login_using_password_confirmation);
+        builder.setCancelable(false);
+        builder.setPositiveButton("Yes", new DialogInterface.OnClickListener() {
+            public void onClick(DialogInterface dialog, int id) {
+                Intent i = new Intent(PassCodeActivity.this, LoginActivity.class);
+                i.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
+                startActivity(i);
+                finish();
+            }
+        })
+                .setNegativeButton("No", null);
+        AlertDialog alert = builder.create();
+        alert.show();
+        Button nbutton = alert.getButton(DialogInterface.BUTTON_NEGATIVE);
+        nbutton.setTextColor(Color.BLUE);
```

Comment:
Hi @rachittt can you change `Color.BLUE` to use the same blue color used in application resources color `primary_dark `

Reply:
sure